ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/lists/ContentProtectionListActivity.java

```diff
@@ -37,6 +36,7 @@ public void onCreate(Bundle savedInstanceState) {
     //Populate the embed map
 //    selectionMap.put("Adobe Pass Integration", new PlayerSelectionOption("VybW5lODrJ0uM9FBo7XTT6TNjTJfr_7G", "B3MDExOuTldXc1CiXbzAauYN7Iui", "http://www.ooyala.com", AdobePassSampleAppAcitivity.class) );
     selectionMap.put("Ooyala Player Token", new PlayerSelectionOption("0yMjJ2ZDosUnthiqqIM3c8Eb8Ilx5r52", "c0cTkxOqALQviQIGAHWY5hP0q9gU", "http://www.ooyala.com", OoyalaPlayerTokenPlayerActivity.class) );
+    selectionMap.put("DRM DASH", new PlayerSelectionOption("Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm", "FoeG863GnBL4IhhlFC1Q2jqbkH9m", "http://www.ooyala.com", OoyalaPlayerTokenPlayerActivity.class));
```

Comment:
Are you able to play this asset? I haven't been able to play it.

Reply:
Yes, I'm able to play this DRM DASH. As far as I know not all of Android devices support DRM so you can check, for example, https://www.ghacks.net/2016/12/03/check-your-android-devices-drm-capabilities/